app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,56 @@
+class CommentsController < ApplicationController
+  before_action :set_project
+  before_action :set_task
+  before_action :set_comment, only: %i[edit update destroy]
+  before_action -> { authorize! @task }
+
+  def new
+  end
```

Comment:
WE DON'T NEED IT AT ALL

Reply:
You don't have route for this action, it is redundant. Do you understand it?

Reply:
I got it